src/Data/Text/Builder/Linear/Array.hs

```diff
@@ -0,0 +1,72 @@
+-- |
+-- Copyright:   (c) 2022 Andrew Lelechenko
```

Comment:
Feel free to add yourself here and elsewhere.

Reply:
Thanks! Done.